engine/src/flutter/txt/src/skia/paragraph_skia.cc

```diff
@@ -120,8 +120,10 @@ class DisplayListParagraphPainter : public skt::ParagraphPainter {
     }
     std::shared_ptr<DlText> text;
     if (impeller_enabled_) {
+#if IMPELLER_SUPPORTS_RENDERING
```

Comment:
Should this have the if inside it as well? Otherwise we might have a mismatch? Or is that possible?

Reply:
I don't think it's possible for that to happen, but I can switch this up to cover the whole if statement.